soc/arm/st_stm32/stm32f1/Kconfig.soc

```diff
@@ -8,6 +8,18 @@ choice
 	prompt "STM32F1x MCU Selection"
 	depends on SOC_SERIES_STM32F1X
 
+config SOC_STM32F100CX
```

Comment:
Replace by XE and XB variants

Reply:
Fixed. Left VX and added ZX as discussed above